ext/windows/run_facter_interactive.bat

```diff
@@ -0,0 +1,8 @@
+@echo Running facter on demand ...

```

Comment:
Should this be followed by `@echo off`? Otherwise we get all the following commands printed out.